mysticeti-core/src/simulated_network.rs

```diff
@@ -55,15 +55,19 @@ impl SimulatedNetwork {
     pub async fn connect(&self, a: usize, b: usize) {
         let (a_sender, a_receiver) = Self::latency_channel();
         let (b_sender, b_receiver) = Self::latency_channel();
+        let (_al_sender, al_receiver) = tokio::sync::watch::channel(Duration::from_secs(0));
+        let (_bl_sender, bl_receiver) = tokio::sync::watch::channel(Duration::from_secs(0));
```

Comment:
Is there an easy way to indicate what these channels are for (e.g comments or more explicit names)?